scripts/installscripts/buster-install-default.sh

```diff
@@ -798,7 +803,7 @@ install_main() {
     ${apt_get} upgrade
 
     # some packages are only available on raspberry pi's but not on test docker containers running on x86_64 machines
-    if [[ $(uname -m) =~ ^armv.+$ ]]; then
```

Comment:
Is there a specific reason to change this?

Reply:
Yes, there is: `^armv.+$` for an architecture pattern has nothing to do with the fact if the platform is a Raspberry Pi or not. At the same time, not even all Raspberry Pis are `arm` architecture; for example, see this output from my dev Raspberry Pi 3B:

```
root@phoniebox-dev:~# uname -m
aarch64
root@phoniebox-dev:~# cat /proc/cpuinfo | tail -n 1
Model		: Raspberry Pi 3 Model B Plus Rev 1.3
root@phoniebox-dev:~#
```

My change brings a way more precise detection.

Reply:
Ok, 👌